maven-java/kalix-maven-plugin/src/main/scala/kalix/RunAllMojo.scala

```diff
@@ -21,8 +21,8 @@ class RunAllMojo extends RunParameters with DockerParameters {
     RunMojo(
       mainClass,
       logConfig,
-      dockerComposeUtils.readUserFunctionPort,
-      dockerComposeUtils.readServicePortMappings,
+      dockerComposeUtils.userFunctionPort,
+      dockerComposeUtils.localServicePortMappings,
```

Comment:
When we configure the locally run service, we use the service port mappings we read from the docker file, but remove any references to hosts. We only need the ports.